extension.ts

```diff
@@ -91,7 +91,9 @@ function initializeExtensionVariables(ctx: vscode.ExtensionContext): void {
     ext.ui = new AzureUserInput(ctx.globalState);
   }
   ext.context = ctx;
-  ext.outputChannel = util.getOutputChannel();
+
+  ext.outputChannel = vscode.window.createOutputChannel("Docker");
```

Comment:
`createOutputChannel()` returns a "disposable"; is that (should that) be added to the disposable list for when the extension is unloaded?

Reply:
Yep